src/decrypt/utils.ts

```diff
@@ -32,13 +32,20 @@ export function BytesHasPrefix(data: Uint8Array, prefix: number[]): boolean {
     })
 }
 
+export function BytesEqual(a: Uint8Array, b: Uint8Array,): boolean {
+    if (a.length != b.length) return false
```

Comment:
虽然 `!=` 和 `!==` 在这里没差啦… 有考虑加入 `eslint` 到项目吗？

```suggestion
    if (a.length !== b.length) return false
```

Reply:
有考虑，正准备有空把prettier和eslint加进来
这个项目是几年前第一次写前端项目，很多历史遗留问题 😂